src/Rule/Callback.php

```diff
@@ -79,11 +79,23 @@ public function afterInitAttribute(DataSetInterface $dataSet): void
             return;
         }
 
-        try {
-            $this->callback = Closure::fromCallable([$dataSet->getObject()::class, $this->method]);
-        } catch (TypeError) {
-            throw new InvalidArgumentException('Method must exist and have public and static modifiers.');
```

Comment:
The check for non-public method is still needed.

Reply:
If it works for private methods, public and protected methods should work also.

Reply:
Missed that it was already presented. Explicit `public` and `protected` tests won't be redundant in my opinion. But yes, maybe private is enough for now.